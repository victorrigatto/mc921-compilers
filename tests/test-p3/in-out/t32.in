int f(int m, int n) {
    if (! (n >= 2)) {
        n = n - 2;
        return n;
    }
    else {
        m = m - 1;
        return m;
    }
}

int main() {
    int m = -9;
    char n = '4';
    print(f(m, n));
    return 0;
}
