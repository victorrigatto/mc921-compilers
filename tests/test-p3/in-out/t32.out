SemanticError: Type mismatch with parameter n @ 15:16
